src/docs/teams/debugger/using-the-debugger.md

```diff
@@ -0,0 +1,77 @@
+
+## Debugger Control Palette
+
+Located directly below the transaction hash is the debugger's control palette.
+
+<figure class="screenshot">
+  <img class="figure-shadow mb-2 w-50" src="/img/docs/teams/debugger-control-palette.png" alt="Teams Debugger Control Palette">
+  <figcaption class="text-center">The Truffle Teams Debugger Control Palette.</figcaption>
+</figure>
+
+From left to right, the controls, with a description of their associated action, are as follows. Note that you can also hover a control to display its associated name.
+
+- <code>Continue</code> - continue until the next breakpoint is reached or the last line is executed
+- <code>Step Over</code> - steps over the current line
+- <code>Step Into</code> - steps into the function call or contract creation currently being evaluated
+- <code>Step Out</code> - steps out of the currently running function
+- <code>Restart</code> - restarts the debugger session
+
+## Debugger Tabs Control
+
+Located directly the debugger control palette are the tabs representing all the files that will be accessed as part of the transaction's execution path. It's worth noting that it's likely your project may contain more files than are displayed here.
+
+<figure class="screenshot">
+  <img class="figure-shadow mb-2 w-50" src="/img/docs/teams/debugger-interface-tabs.png" alt="Teams Debugger Tabs Control">
+  <figcaption class="text-center">The Truffle Teams Debugger Tabs Control.</figcaption>
+</figure>
+
+There are two noteworthy visual cues with the debugger tabs. The first is that of an "open tab", which means that you're looking at the file's code below and is represented with a light green background. The second is that of "active tab" which is represented by an orange circle <i class="fas fa-dot-circle" style="color: #dc9e5b"></i> to the right of the filename (e.g. `TruffleBadge.sol`) which indicates that the debugger is currently paused within this file.
+
+As you use the debugger's controls to step through a transaction you will likely see the active tab update. Note that the open tab will also update when the code steps into it.
+
+## Debugger Code Panel
+
+As the core of the Truffle Teams Debugger is the code panel. This is a read-only display of the code that the transaction will be stepping through as part of its execution. An example is shown in the screenshot below.
+
+<figure class="screenshot">
+  <img class="figure-shadow mb-2 w-50" src="/img/docs/teams/debugger-code.png" alt="Teams Debugger Code Panel">
+  <figcaption class="text-center">The Truffle Teams Debugger Code Panel.</figcaption>
+</figure>
+
+The main thing to note is the active line which is highlighted in yellow. As your step through your transaction this will update accordingly.
+
+## Debugger State Variable Inspector
```

Comment:
the variable inspector includes more than just state variables (using state here implies contract state)

```suggestion
## Debugger Variable Inspector
```